CMSgov/mint-app
# [Easi 4428]/change history order be

# EASI-4428

## Changes and Description

- Update the order type on the translation field GQL types
    - unit test updates for translations
         - field order is not duplicated in a translation
         - Translation fields must be a struct and not a pointer to a struct. (Since a translation is mandatory)
    - update all translations that violated these unit tests
- Add field_order to the translated_audit_field table
   - order results from translated audit field by field_order
- Update the postman collection to include the new field


## How to test this change
1. Seed the data base
2. Translate the data
3. Fetch the data
  a. assert the returned fields
      1. have the new order
      2. are returned in order.
      
 Also, please double check the changes to translations. It should only be changing some fields to be mandatory, and also updating the ordering of fields.


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
